downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -55,6 +55,40 @@ annotation annot on field;
 type T [int, @annot string];
 ```
 
+#### Support for annotations on fields of anonymous record types
```

Comment:
I'm not sure if we should add this to the release note. Given that they
- were already supported at compile-time
- cannot be accessed in Ballerina code at runtime

If at all, I would skip the example and say field annotation values of record type descriptors defined without a type definition can now be accessed at runtime. The current title is misleading since they were supported at compile-time already.

Reply:
I agree with you. My concern is that without the ability to access it at runtime, there is no use in defining annotations. Isn't it? That is why I keep the title as above.

Reply:
Still used/useful at compile-time.

Reply:
+1 to Say, "field annotation values of record type descriptors defined without a type definition can now be accessed at runtime"

Reply:
no need to add an example. right?

Reply:
@MaryamZi @hasithaa

Reply:
Yeah, doesn't require an example. Actually, this is technically a bug fix. Shall we move this to the bug fixes section?